### ISSUE TITLE
Delete inventory objects takes a long time

### ISSUE THREAD

**Issue by DaDomDa** [glpi-project/glpi#15205]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.9

### Bug description

Hello, 

after the update to 10.0.8 the final deletion of inventory objects (computer, monitor, network device, devices, etc.) takes a very long time and fails if the "max_execution_time" is < 30 seconds. The problem persists even after updating to 10.0.9. 
Moving to the recycle bin works without problems. It doesn't matter if the deletion is triggered from the object or via the mass action. Deleting other objects (software, tickets, etc.) works normally.

When observing the database, it is noticeable that several thousands of queries are started per objects on the tables "glpi_ipadresses" and "glpi_networknames". Here are two examples: 

`SELECT * FROM glpi_networknames WHERE glpi_networknames.id = '93152' LIMIT 1`

`SELECT * FROM glpi_ipaddresses WHERE glpi_ipaddresses.id = '27152' LIMIT 1`

### Relevant log output

```shell
[2023-07-12 16:38:50] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: networkports_id in /var/www/glpi/src/CommonDBConnexity.php at line 474
  Backtrace :
  src/CommonDBChild.php:560                          CommonDBConnexity->getItemsForLog()
  src/CommonDBTM.php:1721                            CommonDBChild->post_updateItem()
  src/CommonDBTM.php:939                             CommonDBTM->update()
  src/CommonDBTM.php:788                             CommonDBTM->cleanRelationData()
  src/CommonDBTM.php:2068                            CommonDBTM->deleteFromDB()
  src/CommonDBConnexity.php:127                      CommonDBTM->delete()
  src/CommonDBTM.php:992                             CommonDBConnexity->cleanDBonItemDelete()
  src/CommonDBTM.php:1119                            CommonDBTM->deleteChildrenAndRelationsFromDb()
  src/CommonDBTM.php:789                             CommonDBTM->cleanRelationTable()
  src/CommonDBTM.php:2068                            CommonDBTM->deleteFromDB()
  src/MassiveAction.php:1501                         CommonDBTM->delete()
  src/MassiveAction.php:1414                         MassiveAction::processMassiveActionsForOneItemtype()
  src/MassiveAction.php:1392                         MassiveAction->processForSeveralItemtypes()
  front/massiveaction.php:59                         MassiveAction->process()
  
[2023-07-12 16:38:50] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: networkports_id in /var/www/glpi/src/CommonDBConnexity.php at line 481
  Backtrace :
  src/CommonDBChild.php:560                          CommonDBConnexity->getItemsForLog()
  src/CommonDBTM.php:1721                            CommonDBChild->post_updateItem()
  src/CommonDBTM.php:939                             CommonDBTM->update()
  src/CommonDBTM.php:788                             CommonDBTM->cleanRelationData()
  src/CommonDBTM.php:2068                            CommonDBTM->deleteFromDB()
  src/CommonDBConnexity.php:127                      CommonDBTM->delete()
  src/CommonDBTM.php:992                             CommonDBConnexity->cleanDBonItemDelete()
  src/CommonDBTM.php:1119                            CommonDBTM->deleteChildrenAndRelationsFromDb()
  src/CommonDBTM.php:789                             CommonDBTM->cleanRelationTable()
  src/CommonDBTM.php:2068                            CommonDBTM->deleteFromDB()
  src/MassiveAction.php:1501                         CommonDBTM->delete()
  src/MassiveAction.php:1414                         MassiveAction::processMassiveActionsForOneItemtype()
  src/MassiveAction.php:1392                         MassiveAction->processForSeveralItemtypes()
  front/massiveaction.php:59                         MassiveAction->process()
```

### Steps To reproduce

1. Go to Computer
2. Select several computers and move them to the Recycle Bin
3. Open the recycle bin
4. Delete all objects permanently

Or 

1. Open a computer
2. Move it to the recycle bin
3. Then delete the computer permanently

### Your GLPI setup information

GLPI 10.0.9 (/glpi => /var/www/glpi)
Installation mode: TARBALL
Current language:de_DE

**Comment by stonebuzz** [comment]
Can you try this

https://github.com/glpi-project/glpi/pull/15194

Best regards

**Comment by DaDomDa** [comment]
Hi @stonebuzz 

thank you for your fast reply. 

Problems seems to be solved by the fix. Deleting inventory objects is possible again.

Greetings

**Comment by cedric-anne** [comment]
Fixed by #15194.
